DatadogRUM/Sources/RUMMonitor/Scopes/RUMViewScope.swift

```diff
@@ -523,7 +531,28 @@ internal class RUMViewScope: RUMScope, RUMContextProvider {
         let refreshRateInfo = vitalInfoSampler?.refreshRate
         let isSlowRendered = refreshRateInfo?.meanValue.map { $0 < Constants.slowRenderingThresholdFPS }
         let networkSettledTime = networkSettledMetric.value(with: context.applicationStateHistory)
-        let interactionToNextViewTime = interactionToNextViewMetric.value(for: viewUUID)
+        var interactionToNextViewTime = interactionToNextViewMetric.value(for: viewUUID)
+        // Only look for an internal override attribute if native INV calculations are disabled
+        if interactionToNextViewTime == .failure(.disabled),
+           let customInvValue = internalAttributes[RUMInternalAttributes.customINVValue] as? (any BinaryInteger) {
+            let customInvValue = Int64(customInvValue)
+            interactionToNextViewTime = .success(TimeInterval(fromNanoseconds: customInvValue))
+        }
+
+        // Only add the performance member if we have a value for it
+        let performance: RUMViewEvent.View.Performance?
+        if let fbcMetric = internalAttributes[RUMInternalAttributes.flutterFirstBuildComplete] as? (any BinaryInteger) {
+            let fbcMetric = Int64(fbcMetric)
```

Comment:
blocking/ Needs overflow check:
```suggestion
            let fbcMetric = Int64(withNoOverflow: fbcMetric)
```